apps/institution-service/src/app.ts

```diff
@@ -14,6 +15,7 @@ app.use(logger("dev"));
 
 // Routes
 app.use("/institutions", institutionRoutes);
+app.use(permissionsRoutes);
```

Comment:
I think it's kind of nice to put the base route here just because then in the routes file you don't have to include `/permissions` in each route you define. You only have one route in permissionsRoutes so it's not repetitive in this case.

Reply:
It also limits the scope of permissionRoutes file. Otherwise you could technically add institution endpoints or anything else in there too.

Reply:
Done